resources/puppetlabs/lein-ezbake/template/global/tasks/build.rake

```diff
@@ -11,19 +11,35 @@ namespace :pl do
     pkg_path = '../pkg'
     FileUtils.mv(Dir.glob("pkg/*.gz").join(''), FileUtils.pwd)
     # unpack the tarball we made during the build step
-    `tar xf #{Dir.glob("*.gz").join('')}`
+    stdout, stderr, exitstatus = Pkg::Util::Execution.capture3(%(tar xf #{Dir.glob("*.gz").join('')}))
+    Pkg::Util::Execution.success?(exitstatus) or raise "Error unpacking tarball: #{stderr}"
     Dir.chdir("#{Pkg::Config.project}-#{Pkg::Config.version}") do
       Pkg::Config.final_mocks.split(" ").each do |mock|
         platform = mock.split('-')[1..-2].join('-')
         platform_path = platform.gsub(/\-/, '')
         os, ver = /([a-zA-Z]+)(\d+)/.match(platform_path).captures
-        platform_path = "#{os}/#{ver}"
-        `bash controller.sh #{os} #{ver}`
-        # carry forward defaults from mock.rake
-        repo = Pkg::Config.yum_repo_name || 'products'
+        puts "===================================="
+        puts "Packaging for #{os} #{ver}"
+        puts "===================================="
+        stdout, stderr, exitstatus = Pkg::Util::Execution.capture3(%(bash controller.sh #{os} #{ver}))
+        Pkg::Util::Execution.success?(exitstatus) or raise "Error running packaging: #{stdout}\n#{stderr}"
+        puts "#{stdout}\n#{stderr}"
+
+        # I'm so sorry
+        # These paths are hard-coded in packaging, so hard code here too.
+        # When everything is moved to packaging 1.0.x this should be able
```

Comment:
It looks like even moving to 1.0.x we don't get to fix this :( However, moving to artifactory, we will!

Reply:
ok, cool. I can update the comment